### ISSUE TITLE
Bug: google.visualization object not complete

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#1314]
```
What steps will reproduce the problem? Please provide a link to a
demonstration page if at all possible, or attach code.

In a normal browser everything works great, but when I test it on a HbbTv browser (Like
the one that Humax has)
it breaks. The visualization object only has 3 keys. It breaks on the NumberFormat
function (4th key).


What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)? CoreChart


Are you using the test environment (version 1.1)?
(If you are not sure, answer NO)
NO

What operating system and browser are you using?
Linux system, with Opera 9.8 (HbbTv)



*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `jeroen@appstrakt.com` on 2013-09-16 06:06:20

**Comment by orwant** [comment]
```
Got some more info:
- The issue occurs with the Opera SDK 3.3 and lower
- This is the error that I get: Syntax error at line 241 while loading: expected ')',
got keyword 'in'

```

Original issue reported on code.google.com by `jeroen@appstrakt.com` on 2013-09-16 15:40:39

**Comment by orwant** [comment]
```
I just found out that the problem is fixed in the 1.1.
```

Original issue reported on code.google.com by `jeroen@appstrakt.com` on 2013-09-17 06:04:14

**Comment by orwant** [comment]
```
That's good to learn.  Thanks for reporting back.
```

Original issue reported on code.google.com by `dlaliberte@google.com` on 2013-09-17 12:46:00
